src/System.Diagnostics.Process/src/System/Diagnostics/Process.UnknownUnix.cs

```diff
@@ -77,5 +77,12 @@ private static string GetExePath()
         {
             throw new PlatformNotSupportedException();
         }
+
+        /// <summary>Gets execution path</summary>
+        private string GetPathToOpenFile()
```

Comment:
This doesn’t seem to be used since TargetsUnknownLinux will no longer be set to true, and you’re getting rid of the Unix osgroup build for this guy.

Reply:
I did not want to delete the UnknownUnix files at this moment @safern. They can still be used as initial stub in future.